src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -1417,7 +1431,7 @@
   - name: Creates a secret using the specified certificate version.
     text: >
         az afd secret create -g group --profile-name profile --secret-name secret1 --secret-version version1
-        --secret-source /subscriptions/sub1/resourceGroups/rg1/providers/Microsoft.KeyVault/vaults/vault1/certificates/cert1
```

Comment:
Can 'certificate' still work or customer has to use 'secrets'?

Reply:
Customer could use either certificates or secrets with content-type like "application/x-pem-file" or " "application/x-pkcs12", but RP side only accept Resource Reference with "secret". (I am also confused with that design though).   The certificate could also be accessed using the secret reference id, as specified in: https://docs.microsoft.com/en-us/azure/key-vault/certificates/about-certificates

![image](https://user-images.githubusercontent.com/61817681/160072864-65b6d72d-b5e0-4a03-ac9b-d7bbc7ffbdbb.png)